packages/@tinacms/toolkit/src/packages/fields/components/ImageUpload/ImageUpload.tsx

```diff
@@ -22,25 +23,29 @@ interface ImageUploadProps {
   loading?: boolean
 }
 
+const isStringImage = (src: string) => {
```

Comment:
Can we use this existing function for this?
https://github.com/tinacms/tinacms/blob/a464f90e3e4e0bb6da0a716979807166a6f30a32/packages/@tinacms/toolkit/src/components/media/utils.ts#L22

@logan-anderson modified the Regex to work with Tina Cloud images, so not sure if that's something we'll need here or not

Reply:
Ah! I see. This was mainly a quick stab at improving the `src && (src.includes('png') || src.includes('svg'))` logic.